wire-ios/Wire-iOS/Sources/UserInterface/ConversationList/Container/ConversationListViewController.swift

```diff
@@ -98,6 +98,9 @@ final class ConversationListViewController: UIViewController {
 
     required init(viewModel: ViewModel) {
         self.viewModel = viewModel
+        defer {
+            viewModel.viewController = self
+        }
```

Comment:
Using `defer` always makes my brain struggle a bit, because it changes the order of code execution...
I do understand it make sense to always close a file and also when an exception is thrown etc... but here a simple move to line 131 achieves the same result, right?